base/gmp.jl

```diff
@@ -553,6 +554,7 @@ binomial(n::BigInt, k::Integer) = k < 0 ? BigInt(0) : binomial(n, UInt(k))
 ==(x::BigInt, f::CdoubleMax) = isnan(f) ? false : cmp(x,f) == 0
 ==(f::CdoubleMax, x::BigInt) = isnan(f) ? false : cmp(x,f) == 0
 iszero(x::BigInt) = x.size == 0
+isone(x::BigInt) = x == ONE
```

Comment:
I get (`Clong`) 4.321ns, (`ONE`) 4.452ns, (`Culong`) 3.753ns. I agree that it's a very low price to pay for 10-20%.